packages/manager/apps/pci-savings-plan/src/components/SimpleTile/SimpleTile.tsx

```diff
@@ -1,28 +1,24 @@
-import { ODS_THEME_COLOR_INTENT } from '@ovhcloud/ods-common-theming';
-import { ODS_TILE_VARIANT } from '@ovhcloud/ods-components';
-import { OsdsTile } from '@ovhcloud/ods-components/react';
+import { OdsCard } from '@ovhcloud/ods-components/react';
 import React from 'react';
 
 const SimpleTile: React.FC<React.PropsWithChildren<{
   onClick?: () => void;
   isActive?: boolean;
-}>> = ({ children, onClick, isActive }) => (
-  <OsdsTile
-    rounded
-    inline
-    color={
-      isActive ? ODS_THEME_COLOR_INTENT.primary : ODS_THEME_COLOR_INTENT.default
-    }
-    variant={ODS_TILE_VARIANT.stroked}
-    className={`flex items-center justify-center w-1/2 shrink-0 md:shrink md:w-1/4  mr-5 text-center ${
+  className?: string;
+}>> = ({ children, onClick, isActive, className }) => (
+  <OdsCard
+    className={`flex items-center flex-col justify-center w-1/2 shrink-0 md:shrink md:w-1/4  mr-5 text-center p-3 ${
```

Comment:
I think you should use clsx for class conditions

Also it seems a like a lot of class names, could this be factored ?